src/edu/csus/ecs/pc2/services/web/ICLICSResourceConfig.java

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 1989-2024 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+
+/**
+ * Interface that all API web servers must implement
+ * @author John Buck
+ */
+package edu.csus.ecs.pc2.services.web;
+
+import org.glassfish.jersey.server.ResourceConfig;
+
+import edu.csus.ecs.pc2.core.IInternalController;
+import edu.csus.ecs.pc2.core.model.IInternalContest;
+import edu.csus.ecs.pc2.services.eventFeed.WebServerPropertyUtils;
+
+/**
+ * @author john
+ *
```

Comment:
john who?

Reply:
Haha.  Yes. javadoc using my user name by default.  Fixed in next push.

Reply:
This has been pushed.  Please resolve this comment.